createdisk.sh

```diff
@@ -124,10 +124,21 @@ EOF
 fi
 
 if [ "${ARCH}" == "aarch64" ] && [ ${BUNDLE_TYPE} != "okd" ]; then
-   # Install qemu-user-static-x86 package from fedora koji to run x86 image on M1
+   # Install qemu-user-static-x86 package from fedora-updates repo to run x86 image on M1
    # Not supported by RHEL https://access.redhat.com/solutions/5654221 and not included
    # in any subscription repo.
-   ${SSH} core@${VM_IP} -- "sudo rpm-ostree install https://kojipkgs.fedoraproject.org//packages/qemu/8.2.6/3.fc40/aarch64/qemu-user-static-x86-8.2.6-3.fc40.aarch64.rpm"
+   cat > /tmp/fedora-updates.repo <<'EOF'
+[fedora-updates]
+name=Fedora 40 - $basearch - Updates
+metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-f40&arch=$basearch
```

Comment:
This could be f41 to get longer support. We'll need to remember to update this at least once a year.

Reply:
Will update it